components/Componenten/tabs/tabs.config.yml

```diff
@@ -1,9 +1,14 @@
 notes: |
   Toont navigatie-tabs. Gebaseerd op de Bootstrap `nav-tabs` component.
 
-  Speciale aandacht voor de `.disabled` modifier:
+  Aandachtspunten: 
+    * een link heeft een `aria-expanded` attribute: `true` voor een geselecteerde tab, `false` voor een niet-geselecteerde tab;
+    * een link heeft een `aria-controls` attribute met het `id` van de `div` met de inhoud van de tab;
+    * een `div` met de inhoud van een niet geselecteerde tab heeft een attribute `hidden`.
```

Comment:
Nog iets over `aria-labelledby`, en de `tabindex="0"` op het tabblad zelf (waarom is dat eigenlijk?)